### ISSUE TITLE
In the invite section. Create membership function expecting phone number

### ISSUE THREAD

**Issue by irohan4792** [pranaygoel02/SpotLight#6]
createmembership function expects a phone number starting with + and a number not greater than 15 digits. The error looks like this. Any hack to overcome this. I have attached both the toast and console error.
![image](https://github.com/pranaygoel02/SpotLight/assets/86585260/de9c22b2-a402-4135-bdee-276400964eae)
![image](https://github.com/pranaygoel02/SpotLight/assets/86585260/6886ccd7-585f-4ebb-b895-1c773e6b8ccf)
Help would be appreciated.

**Comment by Sanjay-Khadka** [comment]
Hi @irohan4792 
![image](https://github.com/pranaygoel02/SpotLight/assets/98448471/86eda6a0-d179-4b78-ad2f-8139d0dc77c5)
first pass phone: user.phone key value in createMembership function in userList file 
![image](https://github.com/pranaygoel02/SpotLight/assets/98448471/d2eec3c2-9e1f-401c-8c76-c9cf0ea15c1d)
then add phone param and pass that to createMembership function in CreateMembership.logic.js that should fix the issue.

**Comment by irohan4792** [comment]
Thanks Sanjay...Anyways I created a new collection as I needed to implement an extra feature. I was trying this fix earlier but it wasn't solving the issue. Thanks again

**Comment by Sanjay-Khadka** [comment]
> Thanks Sanjay...Anyways I created a new collection as I needed to implement an extra feature. I was trying this fix earlier but it wasn't solving the issue. Thanks again

no worries

**Comment by irohan4792** [comment]
> > Thanks Sanjay...Anyways I created a new collection as I needed to implement an extra feature. I was trying this fix earlier but it wasn't solving the issue. Thanks again
> 
> no worries

hi sanjay...just needed your help to setup notification collection part. Can you help me with it?

**Comment by Sanjay-Khadka** [comment]
> > > Thanks Sanjay...Anyways I created a new collection as I needed to implement an extra feature. I was trying this fix earlier but it wasn't solving the issue. Thanks again
> > 
> > 
> > no worries
> 
> hi sanjay...just needed your help to setup notification collection part. Can you help me with it?

Yeah Sure what do you need ?

**Comment by irohan4792** [comment]
> > > > Thanks Sanjay...Anyways I created a new collection as I needed to implement an extra feature. I was trying this fix earlier but it wasn't solving the issue. Thanks again
> 
> > > 
> 
> > > 
> 
> > > no worries
> 
> > 
> 
> > hi sanjay...just needed your help to setup notification collection part. Can you help me with it?
> 
> 
> 
> Yeah Sure what do you need ? 
> 
> 

The notification part is not working. I am not able to find out what is going on and also the appwrite collection for that. Can you provide me with a step by step solution

**Comment by Sanjay-Khadka** [comment]
@irohan4792 
![Screenshot from 2024-04-16 16-09-37](https://github.com/pranaygoel02/SpotLight/assets/98448471/0c242ea5-6589-45dc-a15f-e94ecd4d7245)
add these attributes and useId index in the index section 
![image](https://github.com/pranaygoel02/SpotLight/assets/98448471/4aa41a28-f681-47fd-81aa-ef3097686b2f)
in your notification collection see if it works

**Comment by irohan4792** [comment]
> @irohan4792 
> 
> ![Screenshot from 2024-04-16 16-09-37](https://github.com/pranaygoel02/SpotLight/assets/98448471/0c242ea5-6589-45dc-a15f-e94ecd4d7245)
> 
> add these attributes and useId index in the index section 
> 
> ![image](https://github.com/pranaygoel02/SpotLight/assets/98448471/4aa41a28-f681-47fd-81aa-ef3097686b2f)
> 
> in your notification collection see if it works
> 
> 

Okay sure

**Comment by irohan4792** [comment]
> @irohan4792 ![Screenshot from 2024-04-16 16-09-37](https://private-user-images.githubusercontent.com/98448471/322806269-0c242ea5-6589-45dc-a15f-e94ecd4d7245.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMyODE5MTMsIm5iZiI6MTcxMzI4MTYxMywicGF0aCI6Ii85ODQ0ODQ3MS8zMjI4MDYyNjktMGMyNDJlYTUtNjU4OS00NWRjLWExNWYtZTk0ZWNkNGQ3MjQ1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDE1MzMzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU3NGQwNjY1NGQzNmNmZmMwZjAxYzA3ZmQwNDUzMmQxZmQ4MjhhZDNiMjRjNDM5ZjlhZjFkOTViYzRmZDgzODMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.pTvVQKhb-PwMSTuWMORyLEOYHn0FUG_rcJ8yfOYdbSM) add these attributes and useId index in the index section ![image](https://private-user-images.githubusercontent.com/98448471/322806414-4aa41a28-f681-47fd-81aa-ef3097686b2f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMyODE5MTMsIm5iZiI6MTcxMzI4MTYxMywicGF0aCI6Ii85ODQ0ODQ3MS8zMjI4MDY0MTQtNGFhNDFhMjgtZjY4MS00N2ZkLTgxYWEtZWYzMDk3Njg2YjJmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDE1MzMzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdiMmI5NTFlNGI4MDg3M2UyOTdlOWIyNTc2NGJjY2U1Njc2MjVhOWY3NTNmZjM4MzkxMTBkYzhiMWY2NDdjNjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.h5KmUBFEiIQDoY6eawG-X5pfgB_eYo2MrmToBMMh91M) in your notification collection see if it works

i have configured the attributes. Its not working ig what else do i need to do.

**Comment by irohan4792** [comment]
> @irohan4792 ![Screenshot from 2024-04-16 16-09-37](https://private-user-images.githubusercontent.com/98448471/322806269-0c242ea5-6589-45dc-a15f-e94ecd4d7245.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMyODg1NDYsIm5iZiI6MTcxMzI4ODI0NiwicGF0aCI6Ii85ODQ0ODQ3MS8zMjI4MDYyNjktMGMyNDJlYTUtNjU4OS00NWRjLWExNWYtZTk0ZWNkNGQ3MjQ1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDE3MjQwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM2NzI0YjhmNjk4Y2UwMjc4OTM5YjRhYmZhYjNiNmQyNWRjMTgyODFmNzU4ZmE2M2EwYmRkMWQ5ZTg0NjZjZDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.4pMOgEhMAuFfNw36-dvVWEwcRrF5d-yDj2W1mrEXv40) add these attributes and useId index in the index section ![image](https://private-user-images.githubusercontent.com/98448471/322806414-4aa41a28-f681-47fd-81aa-ef3097686b2f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMyODg1NDYsIm5iZiI6MTcxMzI4ODI0NiwicGF0aCI6Ii85ODQ0ODQ3MS8zMjI4MDY0MTQtNGFhNDFhMjgtZjY4MS00N2ZkLTgxYWEtZWYzMDk3Njg2YjJmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDE3MjQwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAyOTdmNGQ4NGRkMThhODkxYzdmNzc4ODJmOTFmOWNjYzNiNzY0ZWRkOTAyMTFmZDliOWEyNTY0Y2QxM2E5ZmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.eNf_tB-qmtHtKVc7Q9cW1KRNUB_cMSDZbl0h8w9glG0) in your notification collection see if it works

can we connect? I am unable to figure it out

**Comment by Sanjay-Khadka** [comment]
@irohan4792  yeah sure, I shouldn't have blindly given the solution without knowing what issues you were facing first can you let me know what you are trying to achieve and some of the details of the issues that you are facing or any errors that you have encountered so far ??.

**Comment by irohan4792** [comment]
> @irohan4792 yeah sure, I shouldn't have blindly given the solution without knowing what issues you were facing first can you let me know what you are trying to achieve and some of the details of the issues that you are facing or any errors that you have encountered so far ??.

i am trying to implement notification function. Can you give me your number so that we can connect on meet?

**Comment by irohan4792** [comment]
> > @irohan4792 yeah sure, I shouldn't have blindly given the solution without knowing what issues you were facing first can you let me know what you are trying to achieve and some of the details of the issues that you are facing or any errors that you have encountered so far ??.
> 
> i am trying to implement notification function. Can you give me your number so that we can connect on meet?

or you can provide your email that would be better.

**Comment by Sanjay-Khadka** [comment]
yeah sure it's the same one that's on my github profile. I am a beginner though i don't know how much of help i can be as i am learning it myself.

**Comment by irohan4792** [comment]
> yeah sure it's the same one that's on my github profile. I am a beginner though i don't know how much of help i can be as i am learning it myself.

no worries. btw your email is not visible to me. Can you share it?

**Comment by Sanjay-Khadka** [comment]
sanjaykhadka@tuicms.edu.np

**Comment by irohan4792** [comment]
> sanjaykhadka@tuicms.edu.np
> 
> 

I've sent you a link on your mail please check

**Comment by irohan4792** [comment]
> [sanjaykhadka@tuicms.edu.np](mailto:sanjaykhadka@tuicms.edu.np)

Are you not active on mails. I was waiting for your reply.

**Comment by Sanjay-Khadka** [comment]
Hi Rohan I've sent you an email please reply.
